Podfile

```diff
@@ -0,0 +1,11 @@
+# Uncomment the next line to define a global platform for your project
+# platform :ios, '9.0'
+
+target 'UsersApp' do
+  # Comment the next line if you don't want to use dynamic frameworks
+  use_frameworks!
+
+  # Pods for UsersApp
+pod 'GoogleSignIn'
```

Comment:
use Swift Package Manager to add dependencies from now on, including this framework